### PR TITLE
Save to a ./saved_stories instead of ./

### PR DESCRIPTION
**Pull request by louisgv**
Related: #88 

This allows easier moving of stories generated.

### CODE REVIEW

File: story/story_manager.py [hunk 1]
```diff
@@ -122,10 +122,15 @@ def load_from_local(self, save_name):
...
+        
+        save_path = "./saved_stories/"
+
+        if not os.path.exists(save_path):
+            os.makedirs(save_path)
 
         story_json = self.to_json()
         file_name = "story" + str(self.uuid) + ".json"
-        f = open(file_name, "w")
+        f = open(save_path + file_name, "w")
```
Comment (ben-bay) [inline story/story_manager.py]: Use `os.path.join` here, instead of `+`

File: story/story_manager.py [hunk 2]
```diff
@@ -138,14 +143,18 @@ def save_to_storage(self):
...
+        save_path = "./saved_stories/"
+
+        if not os.path.exists(save_path):
+            return "Error save not found."
 
         file_name = "story" + story_id + ".json"
         cmd = "gsutil cp gs://aidungeonstories/" + file_name + " ."
         os.system(cmd)
-        exists = os.path.isfile(file_name)
+        exists = os.path.isfile(save_path + file_name)
```
Comment (ben-bay) [inline story/story_manager.py]: Use os.path.join here, instead of +

File: story/story_manager.py [hunk 3]
```diff
@@ -138,14 +143,18 @@ def save_to_storage(self):
...
 
         if exists:
-            with open(file_name, "r") as fp:
+            with open(save_path + file_name, "r") as fp:
```
Comment (ben-bay) [inline story/story_manager.py]: Use `os.path.join` here, instead of `+`

**Comment by ben-bay** [comment]
Thank you for the change, can you target this for `develop` in stead of `master`, then I'll merge this.

**Comment by louisgv** [comment]
@ben-bay I will close this one and create a different PR that target `develop`

**Pull request closed without merging**